ngdo-pro/WCS_Projet3
# Guest reservation

Hey guys, 

Here is the guest reservation,

Please, remember to : 

+ app/console doctrine:migrations:migrate
+ app/console doctrine:fixture:load

To test it : 

+ Log as john@email.com
+ Go to http://localhost:8000/bapteme-de-chef/reserver/ (id of a baptismHasUser you have in your database)
+ Test and check your database to verify that your reservation is registered
